Userland/Utilities/sql.cpp

```diff
@@ -156,6 +169,39 @@ class SQLRepl {
     AK::RefPtr<SQL::SQLClient> m_sql_client { nullptr };
     int m_connection_id { 0 };
     Core::EventLoop m_loop;
+    FILE* m_input_file { nullptr };
+    bool m_quit_when_files_read { false };
+    Vector<String> m_input_file_chain {};
+
+    Optional<String> get_line()
+    {
+        if (!m_input_file && !m_input_file_chain.is_empty()) {
+            auto file_name = m_input_file_chain.take_first();
+            m_input_file = fopen(file_name.characters(), "r");
+            if (m_input_file == nullptr) {
+                warnln("Input file {} could not be opened", file_name);
+            }
+        }
+        if (m_input_file) {
```

Comment:
Hmm so before `LibSQL` became an IPC based service, it would have been able to parse the entire program in one go instead of splitting it by line. Can we instead add an IPC to accept a whole program? (assuming the current IPC infra can't already do that).

Reply:
We could. Bit reluctant to do that in this PR, because it opens some questions, not least how to deal with errors. Let's keep it in mind for a next iteration.

Reply:
To clarify: right now the service assumed that a request is a single statement. It doesn't care about newlines, but I'm not sure what happens if you feed it more than one statement. It wouldn't be hard I guess to make it iterate over all the statements in the request, but again, error handling becomes a point of attention.